examples/vmseries/versions.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_version = ">=0.12.29, <0.16"
```

Comment:
Create a very minimal example. It's the only one which is 0.12.29
compatible, because it does not employ module `depends_on` or module
`for_each` anywhere.